app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -231,18 +263,29 @@ public void showScanButton(boolean show) {
     public void showAttendees(List<Attendee> attendees) {
         // The list is loaded from presenter, so we just need
         // to notify RecyclerView to update the data
-        itemAdapter.set(attendees);
+        itemAdapter.setNewList(attendees, true);
         binding.setVariable(BR.attendees, attendees);
         binding.executePendingBindings();
     }
 
     @Override
     public void updateAttendee(int position, Attendee attendee) {
-        itemAdapter.set(position, attendee);
+        itemAdapter.setNewList(attendeesPresenter.getAttendees(), true);
```

Comment:
If one attendee is updated, is the whole view updated or just this one attendee? You can visually see the difference. If the whole view is updated, better use DiffUtils, it does look file with 1000 attendees, but is not the matter of numbers, just this that there shouldn't be a need to update each view show to register change in one